CHANGELOG.md

```diff
@@ -4,9 +4,6 @@
 
 [Unreleased]: https://github.com/chaostoolkit-incubator/chaostoolkit-datadog/compare/main...jitapichab:chaostoolkit-datadog:feature/datadog_metrics
 
-
-## [Unreleased][]
-
```

Comment:
don't remove this

Reply:
I remove this because I have this block duplicated in the last commut.